src/backends/torch/torchlib.cc

```diff
@@ -967,6 +980,25 @@ namespace dd
           forward_method = ad_mllib.get("forward_method").get<std::string>();
       }
 
+    if (output_params.has("bbox") && output_params.get("bbox").get<bool>())
+      {
+        bbox = true;
+      }
+    if (output_params.has("confidence_threshold"))
```

Comment:
Interesting, we were not using it for classification models then! Then we should/could fix it here or in another tiny PR for classification models ?

Reply:
Will fix in another PR

Reply:
Let's not forget!